pack/parallel/bad_encounter.json

```diff
@@ -0,0 +1,88 @@
+				"back_name": "Out For Blood",
+				"back_text": "Elspeth Baudin attacks each investigator in player order, regardless of their current location <i>(even if she is exhausted)</i>.\nFlip this agenda back over.",
+				"code": "90021",
+				"doom": 8,
+				"double_sided": true,
+				"encounter_code": "bad_blood",
+				"encounter_position": 2,
+				"faction_code": "mythos",
+				"illustrator": "Sebastian Rodriguez",
+				"name": "Hyperborean Blood",
+				"pack_code": "bad",
+				"position": 21,
+				"quantity": 1,
+				"text": "<b>Forced</b> - At the end of the enemy phase, if Elspeth Baudin is ready and there is a memory at her location: Reveal a random token from the chaos bag and place it on that location.\n<b>Forced</b> - If there are 1 or more chaos tokens on a location without a memory: Return them to the chaos bag.\n<b>Forced</b> - If there are chaos tokens on a location with a memory that have a combined value of 6 or more (ignoring +/- and treating each [autofail] and [elder_sign] as a -6): Return them to the chaos bag, and Elspeth Baudin collects that memory.",
+				"stage": 1,
+				"type_code": "agenda"
+		},
+		{
+				"back_name": "Memory of Eternal Conquest",
+				"back_text": "<b>If Agnes collected more memories than Elspeth:</b>\n<blockquote><i>Elspeth falls before you, pain coursing through her body. \"Go on, then. Finish it,\" she hisses through her teeth, glaring up at you with seething hatred. Arcane power crackles in your hand. She is your enemy. Your conquest. Her life, her memories, they are in your hands. You should take what is rightfully yours.</i></blockquote>\n<b>(→R1)</b>\n<hr><b>If Elspeth collected more memories than Agnes:</b>\n<blockquote><i>You fall to your knees, overcome by pain. Elspeth looms over you, her hands crackling with arcane energy. \"You are weak,\" she taunts. \"You are not deserving of this power. But I...\" she grins. \"I will be so much better.\"</i></blockquote>\n<b>(→R2)</b>",
+				"clues": null,
+				"code": "90022",
+				"double_sided": true,
+				"encounter_code": "bad_blood",
+				"encounter_position": 3,
+				"faction_code": "mythos",
+				"flavor": "You must recover the lost memories of your previous life before Elspeth does!",
+				"illustrator": "Joshua Calrós",
+				"name": "A Walk Down Memory Lane",
+				"pack_code": "bad",
+				"position": 22,
+				"quantity": 1,
+				"stage": 1,
+				"text": "[fast]Investigators at Agnes Baker's location spend 2 [per_investigator] clues, as a group: Agnes Baker collects the memory at her location.\n<b>Forced</b> - If Agnes Baker is defeated, proceed to <b>(→R2)</b>\n<b>Objective</b> - Collect more memories than Elspeth. If all 9 memories have been collected <i>(by either party)</i>, advance.",
+				"type_code": "act"
+		},
+    {
+        "code": "90023",
```

Comment:
I believe this code should go the other way.  The front of the card is the enemy, so that card should have the `"back_link": "90023b"`.

Not totally sure, @drawntotheflame do you know?

Reply:
yeah that part was definitely the part i was least certain of, i copied the format for Ichtaca, which I now realize is probably wrong as well? (https://arkhamdb.com/card/04325b shows Yig's Mercy, which feels incorrect)

Reply:
Yeah, front should be the enemy - normally we go with what you see first in the scenario (this is an enemy with story on the back).